extra/exe-builder/Program.cs

```diff
@@ -49,8 +57,9 @@ public class UptimeKumaApplicationContext : ApplicationContext
 
         private RegistryKey registryKey = Registry.CurrentUser.OpenSubKey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Run", true);
 
+        private readonly bool intranetOnly = false;
 
-        public UptimeKumaApplicationContext() {
+        public UptimeKumaApplicationContext(bool intranetOnly = false) {
```

Comment:
```suggestion
        public UptimeKumaApplicationContext(bool intranetOnly) {
```

See above (GitHub won't let me merge this into one suggestion)